engines/bops_api/schemas/odp/v0.6.0/applicationSubmission.json

```diff
@@ -30,6 +30,14 @@
           "format": "datetime",
           "type": "string"
         },
+        "validFrom": {
+          "format": "datetime",
+          "type": ["string", "null"]
+        },
+        "publishedAt": {
+          "format": "datetime",
+          "type": ["string", "null"]
+        },
```

Comment:
Not for now, and a note for me too, but next time we touch the application details let's extract all this into a reusable schema as we use it here and the search